test/unit/css.test.js

```diff
@@ -53,4 +53,29 @@ suite('CSS', function() {
     var testVal = parseFloat(mqF.css('margin-right')) - parseFloat(mqF.css('margin-left'));
     assert.ok(testVal > 0, 'this should be truthy') ;
   });
+
+  test('unary PlusMinus before separator', function () {
+    var mq = MQ.MathField($('<span></span>').appendTo('#mock')[0]);
+    mq.latex('(-1,-1-1)-1,(+1;+1+1)+1,(\\pm{}1,\\pm{}1\\pm{}1)\\pm{}1');
+    var spans = $(mq.el()).find('.mq-root-block').find('span');
+    assert.ok(spans.length >= 35, 'PlusMinus expression parsed incorrectly');
```

Comment:
is the nonstrict equality to not be dependent on the specific spans that parens happen to be implemented with? I think using hardcoded indices later makes that moot right? I'm gonna change this to strict equality